### PR TITLE
Item holding for tiny pumpkins

### PR DESCRIPTION
**Pull request by Juuxel**
tiny pumpkin holding a sword and a murdered potato

### CODE REVIEW

File: src/main/java/com/fabriccommunity/spookytime/registry/SpookyBlocks.java [hunk 1]
```diff
@@ -31,6 +36,8 @@
 	public static Block SPOOKIUM_ORE = register("spookium_ore", new OreBlock(FabricBlockSettings.copy(Blocks.IRON_ORE).build()), new Item.Settings().group(SpookyTime.GROUP).rarity(Rarity.EPIC));
 	public static Block SPOOKIUM_BLOCK = register("spookium_block", new Block(FabricBlockSettings.copy(Blocks.IRON_BLOCK).materialColor(MaterialColor.RED).build()), new Item.Settings().group(SpookyTime.GROUP).rarity(Rarity.EPIC));
 	;
+
+	public static final BlockEntityType<TinyPumpkinBlockEntity> TINY_PUMPKIN_ENTITY = register("tiny_pumpkin", TinyPumpkinBlockEntity::new, TINY_PUMPKIN);
```
Comment (Shrimpstress) [inline src/main/java/com/fabriccommunity/spookytime/registry/SpookyBlocks.java]: This variable name should be changed to `TINY_PUMPKIN_BLOCK_ENTITY` to be clearer that this is not an actual Entity. We will be having an actual Entity that likely uses this same model in the future, so keeping them distinct is quite important.

**Review by Shrimpstress — CHANGES_REQUESTED** [review]
Everything looks great besides one little issue.

File: src/main/java/com/fabriccommunity/spookytime/registry/SpookyBlocks.java (hunk 1 above)
Comment (Juuxel) [reply src/main/java/com/fabriccommunity/spookytime/registry/SpookyBlocks.java]: Fixed.

**Review by Shrimpstress — APPROVED** [review]
This looks good to me now.

**Comment by Shrimpstress** [comment]
This now has some slight merge conflicts due to a recently merged PR.

**Review by vini2003 — APPROVED** [review]
Is good. Is Tiny Pumpkin.

**Pull request merged**